insights/parsers/openstack_server_list_all.py

```diff
@@ -0,0 +1,98 @@
+"""
+openstack server list --all command - Command
+=============================================
+"""
+
+from insights import CommandParser, parser, LegacyItemAccess
+from insights.parsers import ParseException, SkipException, get_active_lines, parse_delimited_table
+from insights.specs import Specs
+
+
+@parser(Specs.openstack_server_list_all)
+class OpenstackServerListAll(LegacyItemAccess, CommandParser):
+    """
+    This parser processes the output of the command `openstack server list --all` and provides
+    the information as a list of dictionary.
+    The LegacyItemAccess class provides some helper functions for dealing with a
+    class having a `data` attribute.
```

Comment:
Whatever you decide to name your attributes, if they are public you should have an `Attributes:` section in your docstring to define them for the developer.  It be better not to use this wording here.